src/settings/BehaviorForm.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+  intlShape,
+  FormattedMessage,
+} from 'react-intl';
+import {
+  Form,
+} from 'react-final-form';
+
+import {
+  Pane,
+  PaneFooter,
+  Select,
+  Button,
+} from '@folio/stripes/components';
+
+import RowComponent from './RowComponent';
+
+const deletedRecordsSupportSelectValues = (formatMessage) => [
+  {
+    value: 'no',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.no' }),
+  },
+  {
+    value: 'transient',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.transient' }),
+  },
+  {
+    value: 'persistent',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.persistent' }),
+  },
+];
+
+const suppressedRecordsProcessingSelectValues = (formatMessage) => [
+  {
+    value: true,
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.suppressedRecordsProcessing.true' }),
+  },
+  {
+    value: false,
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.suppressedRecordsProcessing.false' }),
+  },
+];
+
+const errorsProcessingSelectValues = (formatMessage) => [
+  {
+    value: '200',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.errorsProcessing.200' }),
+  },
+  {
+    value: 'error',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.errorsProcessing.error' }),
+  },
+];
+
+class BehaviorForm extends Component {
+  static propTypes = {
+    label: PropTypes.node.isRequired,
+    intl: intlShape.isRequired,
+  };
+
+  renderFooter = () => {
+    return (
+      <PaneFooter
+        renderEnd={(
+          <Button
+            data-test-behavior-form-button-save
+            type="submit"
+            buttonStyle="primary paneHeaderNewButton"
+            marginBottom0
+          >
+            <FormattedMessage id="stripes-core.button.save" />
+          </Button>
+        )}
+      />
+    );
+  };
+
+  render() {
+    const {
+      label,
+      intl: {
+        formatMessage,
+      },
+    } = this.props;
+
+    return (
+      <Pane
+        defaultWidth="50%"
+        fluidContentWidth
+        paneTitle={label}
+        footer={this.renderFooter()}
+      >
+        <Form
```

Comment:
Why don't you use stripesFinalForm wrapper? This question relates to all forms in this PR.
Also why onSubmit does nothing?

Reply:
Will be migrate to stripesFinalForm in scope of https://issues.folio.org/browse/MODOAIPMH-110. onSubmit event will be added in scope of https://issues.folio.org/browse/MODOAIPMH-110.